extensions/amp-story/1.0/page-advancement.js

```diff
@@ -524,11 +549,27 @@ class ManualAdvancement extends AdvancementConfig {
 
     return (
       inExpandedMode ||
-      (matches(target, INTERACTIVE_EMBEDDED_COMPONENTS_SELECTORS) &&
+      (this.isInClickableContext_(target) &&
         this.canShowTooltip_(event, pageRect))
     );
   }
 
+  /**
+   * @param {!Element} target
+   * @return {boolean}
+   * @private
+   */
+  isInClickableContext_(target) {
+    const context = parseInt(
+      computedStyle(this.win_, target).getPropertyValue(
+        '--i-amphtml-story-context'
+      ),
+      10
+    );
+
+    return context === StoryContextStore.EMBEDDED_COMPONENT;
+  }
+
```

Comment:
This whole `StoryContextStore` is now irrelevant for this PR, can we remove it?

Reply:
Yup, it's the same (we were using `matches(target, INTERACTIVE_EMBEDDED_COMPONENTS_SELECTORS)` before which basically serves the same function). I reverted it.